packages/utils/src/libp2p/addressSorters.ts

```diff
@@ -21,7 +21,7 @@ export function isMultiaddrLocal(multiaddr: Multiaddr): boolean {
         ipFamily = 'IPv6'
         break
       default:
-        throw Error(`Invalid address family in Multiaddr. Got ${family} but expected either '4' or '6'.`)
+        return false
```

Comment:
I think we should better throw an error ... Libp2p might change their API  and throwing an error makes sure that we notice it.

Reply:
That `catch` block catches that error anyway and turns it into `return false` immediately.
If this function is not supposed to throw ever, we should add some logging to the `catch` block at least.